src/test/librbd/test_librbd.cc

```diff
@@ -861,9 +861,9 @@ TEST_F(TestLibRBD, GetId)
     size_t name_len = 0;
     ASSERT_EQ(-ERANGE, rbd_get_name(image, NULL, &name_len));
     ASSERT_EQ(name_len, name.size() + 1);
-    char image_name[name_len];
```

Comment:
I'd just make it `char image_name[4096];` like `id` above (although that would heavily overprovision it -- 64 would be enough for both).